commit 917532574e82292233a38bd30aa1a52f2e9ff6e9
Author: QbProg <tholag@gmail.com>
Date:   Sun May 15 15:01:00 2011 +0200

    Removed some #pragma comment("lib",...)
    In MSVC and Cmake these should come from the Cmake configuration, and not be fixed anymore

commit d654d21db3f5d3fd5c06270359caa4f77b52ea92
Author: QbProg <tholag@gmail.com>
Date:   Tue May 17 00:47:05 2011 +0200

    Cleaned up previous commits:
    - removed pragmas
    - removed NOP cmake instructions

Index: oce/src/Image/Image_PixMap.cxx
===================================================================
--- oce.orig/src/Image/Image_PixMap.cxx
+++ oce/src/Image/Image_PixMap.cxx
@@ -25,10 +25,7 @@
   #include <FreeImagePlus.h>
   #include <Image_PixMap.ixx>
   /* OCC22216 NOTE: linker dependency can be switched off by undefining macro */ 
-  #ifdef _MSC_VER
-  #pragma comment( lib, "FreeImage.lib" )
-  #pragma comment( lib, "FreeImagePlus.lib" )
-  #endif
+  
 #else
   #include <Image_PixMap.ixx>
   #include <fstream>
Index: oce/src/OpenGl/OpenGl_Workspace_5.cxx
===================================================================
--- oce.orig/src/OpenGl/OpenGl_Workspace_5.cxx
+++ oce/src/OpenGl/OpenGl_Workspace_5.cxx
@@ -34,11 +34,6 @@
 /* OCC22218 NOTE: project dependency on gl2ps is specified by macro */
 #ifdef HAVE_GL2PS
   #include <gl2ps.h>
-  /* OCC22216 NOTE: linker dependency can be switched off by undefining macro. 
-     Pragma comment for gl2ps.lib is defined only here. */ 
-  #ifdef _MSC_VER 
-  #pragma comment( lib, "gl2ps.lib" )
-  #endif
 #endif
 
 #include <OpenGl_TextureBox.hxx>
